doc/book/in-depth-guide/first-module.md

```diff
@@ -332,11 +332,11 @@ return [
 
 The above configuration tells the application that the folder
 `module/Blog/view/` has view files in it that match the above described default
-scheme. It is important to note that with this you can not only ship view files
+scheme [[Not clear to me what default scheme is being referred to here. Do you mean the path structure? Could it be stated explicitly?]] . It is important to note that with this you can not only ship view files
 for your module, but you can also overwrite view files from other modules.
 
 Reload your site now. Finally we are at a point where we see something different
-than an error being displayed! Congratulations, not only have you created a
+than an error being displayed! [[Would be good if it said here what you should see, e.g. the Skeleton Application page with Blog\ListController::indexAction() as the header.]] Congratulations, not only have you created a
```

Comment:
Noted; could you suggest wording for us to use, please?

Reply:
"Finally we are at a point where we see something different than an error being displayed! You should see the standard ZF Skeleton Application template page with ```Blog\ListController::indexAction()``` as the ```<h1>``` header."

A screenshot would be good too :-)

Reply:
Rewording sounds good!

I can supply the screenshot when it's time to merge.

Reply:
Great : )